web/static/js/components/thesis-editor.js

```diff
@@ -223,6 +264,44 @@ class ThesisEditor extends React.Component {
     }
   }
 
+  importContent (t, d, page) {
```

Comment:
Don't use `t` and `d` here. Use `contentType` and `content` or whatever `d` is supposed to be.